src/compliment/utils.clj

```diff
@@ -39,6 +39,29 @@
                                 (or skipping (not match)) (recur pi (inc si) true)
                                 match (recur (inc pi) (inc si) false))))))))
 
+
+(defn fuzzy-matches-multi?
```

Comment:
I'd rather not introduce the new function and reuse the regular `fuzzy-matches?`.

Why do you want `/` as a separator here? Can you give an example? I can imagine it is to complete e.g. `::f/bb` to `::foo/bar-baz`, is that correct?

Reply:
I wanted to have `/` as a separator to support [this](https://github.com/alexander-yakushev/compliment/pull/124/files#diff-0284bbd2df7aded088f2315184de757c2ddc6f86add73908df1101e701004451R16).

In our codebase, we have lots of keywords with a long ns part of the keyword before the `/`, so I would like to not have to always write it out and match also this just partially.

Reply:
I see. I guess this is a reasonable desire. Then I'd rather see `fuzzy-matches?` be extended to accept two separators if needed (and if one is passed, then second = first). Making it a list, turning into a set, and checking against a set will slow down thing considerably. Just make it accept two separators and compare against both.

Reply:
Addressed this in f5b8adac6a7909a63dbff33d45b3d8461df91b11